src/Altinn.Platform/Altinn.Platform.Storage/Storage/Repository/SasTokenProvider.cs

```diff
@@ -74,11 +76,12 @@ public async Task<string> GetSasToken(string org)
 
                 _logger.LogInformation($"Getting secret '{secretName}' from '{keyVaultUri}'.");
 
-                sasToken = await _keyVaultWrapper.GetSecretAsync(keyVaultUri, secretName);
+                sasToken.created = DateTime.Now;
```

Comment:
Use UtcNow to be consistent with other logging?

Reply:
I don't think this will ever be exposed outside the class, but I don't mind changing it if we mean that is more consistent.

Reply:
Changed to UtcNow